chia/server/ws_connection.py

```diff
@@ -751,3 +751,21 @@ def get_peer_logging(self) -> PeerInfo:
 
     def has_capability(self, capability: Capability) -> bool:
         return capability in self.peer_capabilities
+
+    def __hash__(self) -> int:
+        return hash(self.peer_node_id)
```

Comment:
@altendky  had a number of comments about this in an older PR here :https://github.com/Chia-Network/chia-blockchain/pull/16328

I think the basic issue was we did have some problems where a peer was both IPv4 and IPv6. You could open the connection to that peer on both interfaces but it has the same node_id, and I believe this caused some problems. This typically doesn't happen on any large network, but on a small network with only a few peers this happens fairly frequently.

Reply:
I'm updated this after guidance from kyle

Reply:
what do you mean by that? That's what I was doing originally, no?